libs/vmst-client/src/lib/vmst-client.module.ts

```diff
@@ -0,0 +1,42 @@
+import { Module, DynamicModule } from '@nestjs/common'
+import fetch from 'isomorphic-fetch'
+import {
+  Configuration,
+  ParentalLeaveApi,
+  PensionApi,
+  PregnancyApi,
+  UnionApi,
+} from '../../gen/fetch'
```

Comment:
Not important, but I'm not a fan of `gen/fetch` (here or elsewhere). Maybe `generated/client`, or maybe `gen` or `generated` is enough.

Reply:
I thought this was some convention, I'll change to `generated/client`.

Reply:
Yeah. Fine to follow conventions. We can take this up later with the group.